src/main/java/ro/sdaboys/restaurantapp/service/ReservationTimeService.java

```diff
@@ -0,0 +1,59 @@
+package ro.sdaboys.restaurantapp.service;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.sdaboys.restaurantapp.dto.ReservationTimeDto;
+import ro.sdaboys.restaurantapp.exception.ReservationTimeNotFoundException;
+import ro.sdaboys.restaurantapp.model.ReservationTime;
+import ro.sdaboys.restaurantapp.repository.ReservationTimeRepository;
+import ro.sdaboys.restaurantapp.validator.ReservationTimeIntervalValidator;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class ReservationTimeService {
+
+
+    private ModelMapper modelMapper;
+    private ReservationTimeRepository reservationTimeRepository;
+    private ReservationTimeIntervalValidator validator;
+
+    @Autowired
+    public ReservationTimeService(ReservationTimeRepository reservationTimeRepository,
+                                  ReservationTimeIntervalValidator validator, ModelMapper modelMapper) {
+        this.reservationTimeRepository = reservationTimeRepository;
+        this.modelMapper = modelMapper;
+        this.validator = validator;
+    }
```

Comment:
We can remove this and addnotate the class with: @AllArgsConstructor(onConstructor_ = @Autowired)